src/l3.c

```diff
@@ -50,7 +50,7 @@ struct l3_entry
     const char *msg;
     uint64_t    arg1;
     uint64_t    arg2;
-};
+} L3_ENTRY;
```

Comment:
Minor change to consistently move to `typedef struct` name with UPPER_CASE struct name convention.